plugins/snowflake/dbt/include/snowflake/sample_profiles.yml

```
@@ -0,0 +1,29 @@
+default:
+  outputs:
+
+    dev: # User-Password config
+      type: snowflake
+      account: [account id]
```

This tends to trip people up:
```suggestion
      account: [account id + region (if applicable)]
```
and below as well, if you agree